modules/sfp_keybase.py

```diff
@@ -0,0 +1,244 @@
+# -*- coding: utf-8 -*-
+# -------------------------------------------------------------------------------
+# Name:         sfp_keybase
+# Purpose:      Spiderfoot plugin to query KeyBase API
+#               to gather additional information about username 
+#
+# Author:      Krishnasis Mandal <krishnasis@hotmail.com>
+#
+# Created:     22/05/2020
+# Copyright:   (c) Steve Micallef
+# Licence:     GPL
+# -------------------------------------------------------------------------------
+
+from sflib import SpiderFoot, SpiderFootPlugin, SpiderFootEvent
+import urllib.request, urllib.parse, urllib.error
+import json
+import re
+class sfp_keybase(SpiderFootPlugin):
+    """Keybase:Footprint,Investigate,Passive:Public Registries::Obtain additional information about target username"""
+
+    opts = {
+    }
+
+    optdescs = {
+    }
+
+    # Tracking results can be helpful to avoid reporting/processing duplicates
+    results = None
+
+    # Tracking the error state of the module can be useful to detect when a third party
+    # has failed and you don't wish to process any more events.
+    errorState = False
+
+    def setup(self, sfc, userOpts=dict()):
+        self.sf = sfc
+        self.results = self.tempStorage()
+
+        for opt in list(userOpts.keys()):
+            self.opts[opt] = userOpts[opt]
+
+    def watchedEvents(self):
+        return ["USERNAME", "LINKED_URL_EXTERNAL"]
+
+    def producedEvents(self):
+        return ["RAW_RIR_DATA", "SOCIAL_MEDIA", "USERNAME",
+            "HUMAN_NAME", "GEOINFO", "BITCOIN_ADDRESS", 
+            "PGP_KEY"]
+
+    def queryUsername(self, qry):
+
+        params = {
+            'usernames': qry.encode('raw_unicode_escape').decode("ascii", errors='replace'),
+        }
+
+        headers = {
+            'Accept' : "application/json",
+        }
+
+        res = self.sf.fetchUrl(
+          'https://keybase.io/_/api/1.0/user/lookup.json?' + urllib.parse.urlencode(params),
+          headers=headers,
+          timeout=15,
+          useragent=self.opts['_useragent']
+        )
+
+        # In this case, it will always be 200 if keybase is queried 
+        # The actual response codes are stored in status tag of the response
+        if not res['code'] == '200':
+            return None
+        
+        # Replacing null with "None"
+        content = json.loads(str(res['content']).replace(":null", ":\"None\""))
```

Comment:
why the `str()` and then replacing null with None? You should be able to just `json.loads(res['content'])`